### PR TITLE
Misc improvements implemented

### LINKED ISSUE #17: Feedback for improvements
https://www.loom.com/share/13d711fdcb2d41db9a92ce2441b72dd0

@mayanksharma4132 _a

### PR DESCRIPTION
**Pull request by mayanksharma4132**
closes #17

### CODE REVIEW

**Comment by mayanksharma4132** [comment]
@chiraggshah _a
Please check.

File: lib/templates/migrations/create_tags_and_taggings.rb [hunk 1]
```diff
@@ -14,6 +14,6 @@ def change
       t.references :taggable, type: :uuid, null: false, index: true, polymorphic: true
       t.timestamps
     end
-    add_index :neeto_tags_taggings, %i[taggable_type taggable_id tag_id], unique: true, name: 'taggings_uniqueness'
+    add_index :neeto_tags_taggings, %i[taggable_type taggable_id tag_id], unique: true, name: 'neeto_tags_taggings_uniqueness'
```
Comment (chiraggshah) [inline lib/templates/migrations/create_tags_and_taggings.rb:17]: Can you update the index in the dummy app as well.

**Review by chiraggshah — APPROVED** [review]
@mayanksharma4132 _a : Added a comment.

**Comment by mayanksharma4132** [comment]
@chiraggshah _a
Please check.

**Comment by mayanksharma4132** [comment]
@chiraggshah _a
Please check.

**Pull request merged**